### DISCUSSION TITLE
Welcome to HackCon VIII Discussions!

### DISCUSSION THREAD

**Discussion by elisemoe** [education/hackconviii#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what brings you to HackCon VIII.

**Comment by mohdsaabith** [comment]
heyy !! happy to be part of this

**Comment by BazilNizam** [comment]
Im so pumped to be a part of this.

**Comment by BazilNizam** [comment]
Which is the best front end technology to learn for webdesigning
